forte/processors/stave/stave_processor.py

```diff
@@ -0,0 +1,302 @@
+        os.environ["DJANGO_ALLOW_ASYNC_UNSAFE"] = "true"
+        django.setup()
+
+        wsgi_app = WSGIContainer(get_wsgi_application())
+        build_path = self.configs.frontend_build_path
+
+        # TODO: Find better logic to deal with routing.
+        #       The following implementation may miss some corner cases.
+        app = Application([
+            url(r"/api/(.*)", self.ProxyHandler, dict(fallback=wsgi_app)),
+            url(r'.*/static/(.*)', StaticFileHandler, {
+                "path": build_path + "/static/"
+            }),
+            url(r'.*/([^/]*\.png)', StaticFileHandler, {"path": build_path}),
+            url(r'.*/([^/]*\.ico)', StaticFileHandler, {"path": build_path}),
+            url(r"/.*", self.ReactHandler, {"build_path": build_path})
+        ])
+
+        server = HTTPServer(app)
+        server.listen(self.configs.port)
+        IOLoop.current().start()
+
+    def _create_project(self, input_pack: DataPack):
+        with requests.Session() as session:
+
+            # Log in as admin user
+            response = session.post(f"{self._url}/api/login",
+                json={
+                    "name": self.configs.user_name,
+                    "password": self.configs.user_password
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+            # Configure and create project
+            if self._project_id < 0:
+                response = session.post(f"{self._url}/api/projects/new",
+                    json={
+                        "type": self.configs.projectType,
+                        "name": self.configs.projectName,
+                        "ontology": self.configs.ontology.__str__(),
+                        "multiOntology": self.configs.multiOntology.__str__(),
+                        "config": self.configs.projectConfigs.__str__()
+                    })
+                logger.info("%d %s", response.status_code, response.text)
+                if response.status_code != 200:
+                    return
+                self._project_id = response.json()["id"]
+
+            # Configure and create document
+            response = session.post(f"{self._url}/api/documents/new",
+                json={
+                    "name": input_pack.pack_name,
+                    "textPack": input_pack.serialize(),
+                    "project_id": self._project_id,
+                })
+            logger.info("%d %s", response.status_code, response.text)
+            if response.status_code != 200:
+                return
+
+    @classmethod
+    def _default_project_configs(cls, ontology: Dict[str, Any]):
+        """
+        Create default project configuration based on ontology.
+        This is translated from JavaScript function `createDefaultConfig` in
+        asyml/stave/src/app/pages/Projects.tsx
```

Comment:
Maybe we create a link to it (also link to the specific commit), since Stave may change and this documentation becomes invalid?

Or if this documentation is not for public usage then it is fine.